cli/src/commands/duplicate.rs

```diff
@@ -70,18 +70,20 @@ pub(crate) struct DuplicateArgs {
     /// commit)
     #[arg(
         long,
-        short,
+        short = 't',
+        visible_alias = "destination",
+        visible_short_alias = 'd',
         value_name = "REVSETS",
-        add = ArgValueCompleter::new(complete::revset_expression_all),
+        add = ArgValueCompleter::new(complete::revset_expression_mutable),
```

Comment:
nit: why did you change this to all mutable revisions? I think all revisions is a better default, since you should be able to rebase your revisions anywhere in the repos history. This is for all such changes in this commit.